apps/vite/src/components/UserSettings.tsx

```
@@ -93,12 +94,12 @@ const UserSettings = ({
             <div className="flex w-1/2 items-center justify-center border-r">
               <span className="sr-only">change language</span>
 
-              {i18n.language == "en" ? (
-                <button onClick={() => i18n.changeLanguage("ne")}>
+              {i18n.language == 'en' ? (
```

Biome

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)